src/BinaryParsers/PEBinary/ProgramDatabase/LinkerCommandLine.cs

```diff
@@ -24,82 +24,160 @@ internal struct LinkerCommandLine
         /// </summary>
         public readonly bool IncrementalLinking;
 
+        /// <summary>
+        /// Whether or not this command line enables Link Time Code Generation (/LTCG)
+        /// </summary>
+        public readonly bool LinkTimeCodeGenerationEnabled;
+
+        /// <summary>
+        /// Whether or not this command line enables Optimize References (/OPT:REF)
+        /// </summary>
+        public readonly bool OptimizeReferencesEnabled;
+
+        /// <summary>
+        /// Whether or not this command line enables Identical COMDAT folding (/OPT:ICF)
+        /// </summary>
+        public readonly bool COMDATFoldingEnabled;
```

Comment:
>COMDATFoldingEnabled [](http://example.com/codeflow?start=29&length=20)

OK, another compromise suggestion, change this casing to 'Comdat' in any API utilization (preserving COMDAT for all user-facing strings/docs.

OR use a more descriptive name, such as:

  CommonDataFoldingEnabled

If you take the latter option, we could build out the summary XML a bit:

Whether or not this command line enables 'Identical Common Data (COMDAT) folding (/OPT:ICF)'

Reply:
Will change